### PR TITLE
Create Drupal 10 page

### LINKED ISSUE #361: Add a Drupal 10 starter config

### PR DESCRIPTION
**Pull request by KeyboardCowboy**
Resolves #361

### CODE REVIEW

File: content/starter-configs/tutorials/drupal-10.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+    # Wait until the mysql service is done building
+    depends: mysql
+
+    # A set of commands to run while building this service
+    commands:
+      # Commands that set up the basic preview infrastructure
+      init:
+        # Bump the memory limit to help Drupal along.
+        - echo "memory_limit = 10240M" >> /usr/local/etc/php/conf.d/my-php.ini
```
Comment (q0rban) [inline content/starter-configs/tutorials/drupal-10.md:76]: Is this necessary? What commands fail without this change?
Also, wow, this bumps the memory limit to 10GB.

**Review by q0rban — CHANGES_REQUESTED** [review]
Thanks so much for working on this! I'd rather leave out the memory bump unless it's strictly necessary for some reason. I'm surprised if so.

File: content/starter-configs/tutorials/drupal-10.md (hunk 1 above)
Comment (KeyboardCowboy) [reply content/starter-configs/tutorials/drupal-10.md:76]: I was having trouble with the site timing out, but I'm wondering if that was a coincidence with other things happening on the server.  Yeah, that should be 1 GB also.  I'll remove that and see what happens.

Comment (KeyboardCowboy) [reply content/starter-configs/tutorials/drupal-10.md:76]: _I must've put a decimal point in the wrong place or something. I always do that. I always mess up some mundane detail._

**Review by q0rban — APPROVED** [review]

**Pull request merged**